src/pages/MonitoringPage.js

```diff
@@ -256,26 +251,19 @@ const Backlog = ({
   const {priority} = priorityState
 
   const [refreshBacklogItems, setRefreshBacklogItems] = React.useState({})
-  const [backlogItems, backlogItemsIsLoading, backlogItemsIsError, setBacklogItems] = useFetchBacklogItems({
+  const [backlogItems, backlogItemsState] = useFetchBacklogItems({
     service: service,
     cfgName: cfgName,
     refresh: refreshBacklogItems,
   })
   const backlogItemsFetchDetails = {
     backlogItems: backlogItems,
-    isLoading: backlogItemsIsLoading,
-    isError: backlogItemsIsError,
+    isLoading: backlogItemsState.isLoading,
+    isError: backlogItemsState.error,
   }
   const [filteredBacklogItems, setFilteredBacklogItems] = React.useState([])
   const [selectedBacklogItems, setSelectedBacklogItems] = React.useState([])
 
-  React.useEffect(() => {
```

Comment:
Is this actually not required anymore?

Reply:
this was a quirky workaround in the first place
instead, useFetch should make sure to clean-up its state on unmount